tools/ebpf/dev_setup.sh

```diff
@@ -3,74 +3,118 @@
 cwd="$(dirname "$0")"
 cd $cwd
 
-set -e
+set -ex
+
+VAGRANT_CWD=$(pwd)
 
 if [[ -f Vagrantfile ]]; then
     echo "detected vagrant file; will clean up"
-    VAGRANT_CWD=$(pwd) vagrant destroy -f
+    vagrant destroy -f
     rm Vagrantfile
 fi
 
 cat <<EOD > Vagrantfile
 Vagrant.configure("2") do |config|
   config.vm.box = "hashicorp-vagrant/ubuntu-16.04"
+  config.vbguest.auto_update = false
   config.vm.synced_folder "$GOPATH/src/github.com/DataDog", "/home/vagrant/go/src/github.com/DataDog"
 end
 EOD
 
-VAGRANT_CWD=$(pwd) vagrant up
+vagrant up
+
+
+echo "installing tools (invoke, clang format, jq, vim)"
+cat <<EOD | vagrant ssh
+sudo apt-get update
+sudo apt-get install -y python-pip unzip curl jq vim clang-format --fix-missing
+sudo pip install invoke pyyaml
+EOD
+
+echo "installing clang and LLVM v8"
+cat <<EOD | vagrant ssh
+wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+echo "deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-8 main" | sudo tee -a /etc/apt/sources.list
+echo "deb-src http://apt.llvm.org/trusty/ llvm-toolchain-trusty-8 main" | sudo tee -a /etc/apt/sources.list
+sudo apt-get update
+sudo apt-get install -y clang-8 llvm-8
+sudo ln -sf /usr/bin/clang-8 /usr/bin/clang
+sudo ln -sf /usr/bin/llc-8 /usr/bin/llc
+EOD
 
-# install invoke
+echo "installing linux-headers"
 cat <<EOD | vagrant ssh
-sudo pip install invoke
+sudo apt-get install -y linux-headers-\$(uname -r)
```

Comment:
I don't think the `\` is suppose to be there right?

Reply:
It's to avoid the expression to be evaluated on the computer running the VM